### PR TITLE
chore(ui): replace nwb by rollup + jest

### PR DESCRIPTION
**Pull request by lionelB**
- remplace nwb par rollup + jest pour les test
- bump de docz
- small fixes

### CODE REVIEW

File: packages/code-du-travail-frontend/src/support/__tests__/__snapshots__/Support.test.js.snap [hunk 1]
```diff
@@ -55,7 +55,7 @@ exports[`<Support /> should render 1`] = `
               viewBox="0 0 20 20"
             >
               <title>
-                Recherche par adresse
+                Rechercher par adresse
```
Comment (douglasduteil) [inline packages/code-du-travail-frontend/src/support/__tests__/__snapshots__/Support.test.js.snap:58]: can be a separate PR :smiling_imp: 
(don't do it if you're tired of git. It's ok if you leave it here)

File: packages/code-du-travail-ui/src/__tests__/BreadCrumbs.test.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+import { render } from "react-testing-library";
+import BreadCrumbs from "../BreadCrumbs";
+
+describe("<Alert />", () => {
+  test("should render with default props", () => {
+    const { container } = render(<BreadCrumbs />);
+    expect(container).toMatchSnapshot();
+  });
+  test("should render width custom props", () => {
+    /* eslint-disable-next-line react/jsx-key */
```
Comment (douglasduteil) [inline packages/code-du-travail-ui/src/__tests__/BreadCrumbs.test.js:11]: why ?
Can use a `//` comment if it's just this line

**Review by douglasduteil — CHANGES_REQUESTED** [review]
:heart: this PR
This will make build faster and bump the coverage over :100:  :)

File: packages/code-du-travail-ui/src/__tests__/BreadCrumbs.test.js (hunk 2 above)
Comment (lionelB) [reply packages/code-du-travail-ui/src/__tests__/BreadCrumbs.test.js:11]: since there is JSX inside an array, it trigger the eslint rule `react/jsx-key`  
ok to change the comment

File: packages/code-du-travail-frontend/src/support/__tests__/__snapshots__/Support.test.js.snap (hunk 1 above)
Comment (lionelB) [reply packages/code-du-travail-frontend/src/support/__tests__/__snapshots__/Support.test.js.snap:58]: I couldn't resist ;)

**Comment by lionelB** [comment]
🆙

File: packages/code-du-travail-frontend/src/support/__tests__/__snapshots__/Support.test.js.snap (hunk 1 above)
Comment (douglasduteil) [reply packages/code-du-travail-frontend/src/support/__tests__/__snapshots__/Support.test.js.snap:58]: So are you leaving it here ?

Comment (lionelB) [reply packages/code-du-travail-frontend/src/support/__tests__/__snapshots__/Support.test.js.snap:58]: yes

**Comment by lionelB** [comment]
@douglasduteil is it ok for you  ?

**Review by douglasduteil — APPROVED** [review]
